### PR TITLE
Update docker setup guide - Closes 1647

### LINKED ISSUE #1647: Update Docker setup guide
### Description

The Docker setup guide needs to be updated for version 0.7

### Location

`setup/docker.adoc`

### Additional Information

https://github.com/LiskHQ/lisk-service

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
### What was the problem?
Closes #1647

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 1]
```diff
@@ -16,54 +17,67 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+====
+*Pre-requisite*
 
-Lisk Service is installed in addition to a Lisk blockchain application such as Lisk Core, in order to provide enriched network data for third-party services.
+Lisk Service is a web application middleware that allows interaction with various blockchain networks based on the Lisk protocol.
+It is recommended to set up a blockchain node first, before setting up Lisk Service.
 
-The first step is therefore to install Lisk Core.
+* To set up a Lisk-Core node, see any of our setup guides, such as xref:{url_core_setup_npm}[].
+* Alternatively, you can setup up a node for a sidechain, as described in the xref:{url_build_blockchain}[] section.
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:41]: Imho link to the developer guides are not fitting here. Probably, a direct link to the app-registry would be cool here, once it is available.

For now, maybe you can just link to the sidechain definition https://lisk.com/documentation/beta/understand-blockchain/interoperability/index.html#mainchain-sidechains

```suggestion
* Alternatively, you can setup up a sidechain node to connect to Lisk Service.
```

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 2]
```diff
@@ -136,51 +150,53 @@ MacOS::
 
 Ensure the port which allows external services communication with the Lisk Service APIs is open.
 
-- 9901: Default API port (HTTP & WebSocket)
+- The *default* API port for *HTTP* & *WebSocket* is: `7887`
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:153]: I don't think the port changed

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 3]
```diff
@@ -211,11 +236,12 @@ make up
 == Stopping Lisk Service
 You can stop Lisk Service again with the following command:
 
+.Working directory: /lisk-service/
 [source,bash]
 ----
 make down
 ----
 
 More commands about how to manage Lisk Service are described on the xref:{url_management}[Docker commands] page.
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:245]: Docker commands hasn't been updated, yet. Please comment out the link to the page.

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 4]
```diff
@@ -191,18 +207,27 @@ vim .env
 [source,bash]
 ----
 # Lisk ecosystem configuration
-LISK_CORE_WS=ws://host.docker.internal:8080 #<1>
+LISK_APP_WS=ws://host.docker.internal:7887 
 ----
 
-<1> The example assumes that the Lisk Core node is running on the host machine, and not inside of a Docker container.
-When running Lisk Core inside of a Docker container, the variable needs to refer to the container: `LISK_CORE_WS="ws://<your_docker_container>:8080"`.
+The aforementioned example assumes that the blockchain node e.g. Lisk Core node is running on the host machine, and not inside of a Docker container.
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:213]:
```suggestion
The `example.env` assumes that the blockchain node e.g. Lisk Core node is running on the host machine, and not inside of a Docker container.
```

I would also move this statement above the code snippet.

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Review by Tschakki — APPROVED** [review]
LGTM 🎉

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 5]
```diff
@@ -16,54 +17,67 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+====
+*Pre-requisite*
 
-Lisk Service is installed in addition to a Lisk blockchain application such as Lisk Core, in order to provide enriched network data for third-party services.
+Lisk Service is a web application middleware that allows interaction with various blockchain networks based on the Lisk protocol.
+It is recommended to set up a blockchain node first, before setting up Lisk Service.
 
-The first step is therefore to install Lisk Core.
+* To set up a Lisk-Core node, see any of our setup guides, such as xref:{url_core_setup_npm}[].
+* Alternatively, you can setup up a sidechain node to connect to Lisk Service.
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:41]: Could you also please add a note to emphasize that setting the `system.keepEventsForHeights: -1` in the config is very important when setting up the node?

```suggestion
* To set up a Lisk-Core node, see any of our setup guides, such as xref:{url_core_setup_npm}[].
* Alternatively, you can setup up a sidechain node to connect to Lisk Service.
* Please make sure to set `system.keepEventsForHeights: -1` in the node config before synchronizing the node with the network.
```

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 6]
```diff
@@ -148,39 +167,41 @@ sudo ufw allow 9901
...
+git checkout v0.7.0
 ----
 
 == Build Lisk Service
 
-Build Lisk Service from local files:
+To build Lisk Service from local files, first, navigate to the `lisk-service` repository on your system:
 
-./lisk-service/
+.Working directory: /lisk-service/
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:182]: Using root directory maybe a wrong assumption. Maybe use the relative path?
```suggestion
.Working directory: ./lisk-service/
```
or
```suggestion
.Working directory: <download_directory>/lisk-service/
```

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 7]
```diff
@@ -191,18 +212,27 @@ vim .env
...
 # Lisk ecosystem configuration
-LISK_CORE_WS=ws://host.docker.internal:8080 #<1>
+LISK_APP_WS=ws://host.docker.internal:7887 
 ----
 
-<1> The example assumes that the Lisk Core node is running on the host machine, and not inside of a Docker container.
-When running Lisk Core inside of a Docker container, the variable needs to refer to the container: `LISK_CORE_WS="ws://<your_docker_container>:8080"`.
+The `example.env` assumes that the blockchain node e.g. Lisk Core node is running on the host machine, and not inside of a Docker container.
+
+When running a blockchain node inside of a Docker container, the following variable needs to refer to the container:
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:220]:
```suggestion
When running a blockchain node inside of a Docker container, the following variable needs to refer to the container. For this to work, the container running the blockchain node and the Lisk Service Blockchain Connector container must share the same docker network:
```

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 8]
```diff
@@ -191,18 +212,27 @@ vim .env
...
+LISK_APP_WS="ws://<your_docker_container>:7887"
+----
 
 TIP: For a complete list of supported environment variables check the xref:{url_references_config}[].
 
 == Starting Lisk Service
 
 Run the following command to start Lisk Service:
 
+.Working directory: /lisk-service/
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:235]: Same as above.

File: docs/antora/modules/ROOT/pages/setup/docker.adoc [hunk 9]
```diff
@@ -211,11 +241,12 @@ make up
 == Stopping Lisk Service
 You can stop Lisk Service again with the following command:
 
+.Working directory: /lisk-service/
```
Comment (sameersubudhi) [inline docs/antora/modules/ROOT/pages/setup/docker.adoc:244]: Same as above.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Please consider the following.

**Review by sameersubudhi — APPROVED** [review]
Thank you for the changes. LGTM 👍

**Pull request merged**